install.sh

```diff
@@ -194,16 +200,16 @@ for package in $BOOTSTRAP_PACKAGES; do
   # This is really ugly, FIXME after #7082 is merged.
```

Comment:
Obviously this isn't something you have to do, but if you wanted you could change the code to not have two copies of each url based on the tar compression, and instead use the value of binary_compression now that that's a possibility.

Reply:
If not, I'll get around to it soon.